### ISSUE TITLE
Feedback for G2C Module 06

### ISSUE THREAD

**Issue by amthomas46** [microsoft/sqlworkshops#92]
06 - What to use When

---  

Spelling/Grammar/Links:  

Intro section:  
* 04 Proof-Of-Concept (POC): After the optimal solution technologies and processes are selected, a POC is set up **wiht** a small **representeative** example of what a solution might look like, as much as possible. If available, a currently-running solution in a **prallel** example can be used  
05 Implementation: Implementing a phased-in rollout of the completed solution **basedd** on  
  
* First sentence, 'in on-premises and in-cloud'? remove first 'in'?  
* "In production, there are normally 6 phases to create a solution = these can be done in-person, or through recorded documents:" replace the = with ; or new sentence?  

6.1 section:  
* 4th paragraph **requirments** to be requirements  
* In activity first paragraph, **descritpions** to be descriptions  
* In project goals for AdventureWorks, **whereever** to **wherever**  
* In project goals for Contoso, **avaialable** to available  
* In WWI description last paragraph before Project Goals **chilli chocolates** to **chili chocolates**  
* For Contoso description, remove the second " The Contoso Corporation is" in the first sentence.

6.3 section:  
* Second paragraph, **mutliple** to multiple  
  
6.4 section:  
* First paragraph, **oftem** to often  
* Last step, **soluition** to solution  

References section:  
* **Azure Pricing Calulator** to calculator  

Links:  
* Checked all links only one mis-points  
* In references at the bottom, the Cloud adoption framework points to the "How to write a problem statement". Should point here: https://docs.microsoft.com/en-us/azure/architecture/cloud-adoption/  

---  
* Feedback: For AW project constraints, I would reword the second bullet: "In some cases, no code change is possible" to something more explicit like "In some case, they are not able or willing to make any code changes". "no code change is possible", to me, is confusing.  

* In 6.2 steps, it reads "Open your ADS document and ...", but I don't think "ADS" has been introduced yet and may be confused with Azure Data Studio or Azure Data Services?  

* I like the decision matrix activity!  

* There are two "workshops" with problems and solutions in https://aka.ms/azuresqlslides that contain "Common Security Scenarios and how to solve them" and "Designing available, scalable, performant apps with SQL DB" (MI and Singleton for both). Somehow I think we should reference these, maybe as an alternative or a deeper architectural session?

**Comment by BuckWoody** [comment]
Thanks for the review! Very helpful. For the last comment, I would love to do that, but I think time is just going to be a factor here. It may make sense to link these assets in Module 5. I hesitate pointing to PPT's on a OneDrive since those go stale quickly, and PPT doesn't often have good context. I wonder if there is a recording of that session that might be more interesting.
